src/App.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+
+import Products from "./routes/Products";
+import AddProduct from "./routes/AddProduct";
+import Home from "./routes/Home";
+import LeftPanel from "./components/LeftPanel";
```

Comment:
Interesting that this is the only thing called on the App that is actually a component, the rest being routes.

Reply:
Yeah, the Routes are components as well, i just figured it would be nice to separate them into their own folder, not convention or anything, but i think it's handy to distiguish.